tracer/src/Datadog.Trace/TraceContext.cs

```diff
@@ -91,6 +91,27 @@ public void AddSpan(Span span)
             }
         }
 
+        public void CloseServerlessSpan()
```

Comment:
If you are not using `CloseServerlessSpan()` anymore, can we remove it please?

Reply:
✅